config/rbac/metrics_service_role.yaml

```diff
@@ -0,0 +1,8 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: controller-manager-metrics-service
+rules:
+ - apiGroups: [""]
```

Comment:
Please use the same format as the rbac yamls for uniformity.

Reply:
Initially thought we require a cluster wide permissions, but moved it to Role now

Reply:
All the yamls are moved to same format